Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/auditrepository/AuditRepositoryLoggerTest.java

```diff
@@ -72,28 +67,22 @@ public void tearDown() {
     }
 
     /**
-     * Test of logNhinPatientDiscAck method, of class AuditRepositoryLogger.
+     * Test of logFindAuditEvents method, of class AuditRepositoryLogger.
      */
     @Test
-    public void testLogNhinPatientDiscAck() {
-        System.out.println("testLogNhinPatientDiscAckServiceEnabled");
+    public void testLogFindAuditEvents() {
+        System.out.println("testLogFindAuditEvents");
```

Comment:
This log statement isn't really adding any value (plus it's a system out call....) -- can you remove this?